index.js

```diff
@@ -53,12 +65,12 @@ function htmlParse(value, allow) {
     value,
     unified()
       .use(html)
-      .use(rehype2retext, text)
+      .use(rehype2retext, makeText())
```

Comment:
Should `config` be passed here too, so that HTML supports profanitySureness as well?